Python_DataEngineering/README.md

```diff
@@ -1,51 +1,38 @@
-* Document list of assumptions made, which are not covered in the stories but have been used in building the app.
-* Ask us anything – hiring at wealth42 dot com.
-
-
-### Assessment Criteria
-1. Maintainability and organisation of code
-2. Correctly choosing normalisation or denormalisation where applicable 
-3. Balancing between speed and quality of code - we understand that candidates can only spend limited time on these challenges.
-If you do make any trade-offs in favour of saving time, do mention them in comments.  
-4. Ownership - going beyond the exact job description to deliver a finished product.
-For a Python/Data Engineer dev, this could potentially translate to: 
-    1. Ensuring safety of API keys (if any are used)
-    2. Creating a query utility that lets a user query the number of free bikes currently available by giving a subset of the 'commonName' string
-5. How much the candidate stays in touch with us during the process of the project
-6. [Guidelines](https://gist.github.com/turbo/efb8d57c145e00dc38907f9526b60f17) for clean commit messages
-
-### Submission Process
-1. Clone this repository
-2. Remove the assignment README’s (and all folders) and start developing your project in it
-3. Add a readme that details on exactly how to run the code
-4. Raise a Pull Request
-
-
-### Learning Resources
-1. [Requests Library Official Documentation](https://requests.readthedocs.io/en/master/)
-2. [RealPython guide to using python requests](https://realpython.com/python-requests/)
-3. [Flask Mega Tutorial](https://blog.miguelgrinberg.com/post/the-flask-mega-tutorial-part-i-hello-world)
-4. [SQLAlchemy Tutorial](https://auth0.com/blog/sqlalchemy-orm-tutorial-for-python-developers/)
-5. [Crontab Tutorial](https://stackabuse.com/scheduling-jobs-with-python-crontab/)
\ No newline at end of file
+Requirements:
+-------------------
+1.Python Compiler
+2.MySql Server
+3.Packages used in this assignment:mysql connector,schedule,datetime,requests.
+
+How To Run:
+-----------------------------------------------------------
+
+1.In project.py file in main method database configuration method,creating tables method are called.scheduler module is used 
+to schedule the script every 5 minutes.
+
+2.Methods defined:
+
+	1.database_config()- it configures the database connectivity and returns db,mycursor references.
+	
+	2.create_table(db,mycursor)-it creates tables[bikes_available1(dock_point,no_of_bikes,timestamp,date),
+	timestamps_table1(timestamp,nof_of_bikes,dock_point,date)]
+
+	3.api()- This method requests the api,and converts into json format and calls the methods to find  nof bikes 
+		available in dockpoints ,as well as in time stamps.
+		This method aslo calls a method which  inserts these required values into database.
+		This method also calls query_from_users to allow users to fetch from database.
+	
+	4.bike_availability_at_doctpoint(list_of_json)-This method returns  no of available bikes per particular dockpoint 

```

Comment:
list_of_json is not clear here - could you write the functions in the format:
`function_name(argument): <Return Type>`